src/KafkaFlow.Admin/Messages/ConsumerMetric.cs

```diff
@@ -0,0 +1,55 @@
+namespace KafkaFlow.Admin.Messages
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// A message that contains data related to consumers partition assigment
+    /// </summary>
+    [DataContract]
+    public class ConsumerMetric
```

Comment:
ConsumerTelemetry maybe?

Reply:
IDK, in my opinion, ConsumerMetric (the metrics of consumers) fits better... but if you want we can change it.

Reply:
done